packages/rancher-webhook/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.2.6-rc3/rancher-webhook-0.2.6-rc3.tgz
-version: 1.0.5
+url: https://github.com/rancher/webhook/releases/download/v0.2.6-rc5/rancher-webhook-0.2.6-rc5.tgz
+version: 1.0.6
```

Comment:
Looks like we have not released a 1.0.5+upX.X.X, so is there a reason the version was bumped again? The last rc was 1.0.5+up0.2.6-rc3 so this should remain 1.0.5 (unless I'm missing something specific to webhook).